mailpoet/assets/js/src/automation/editor/components/icons/plus.tsx

```diff
@@ -0,0 +1,14 @@
+export function PlusIcon(): JSX.Element {
```

Comment:
would it make sense to use @wordpress/icons to keep icons consistent with WordPress?

Reply:
Do you mean the simple icons like "+" specifically or "all" icons? I think we won't find suitable icons for "delay" and things like that but I just added [a commit](https://github.com/mailpoet/mailpoet/pull/4131/commits/4c7b13f981dab333d8c5172ade49e999adc6aad5) to reuse the plus icon as I was able to get it very close to the design using the correct size and fill.

Reply:
> I was able to get it very close to the design

That's right, the PR is probably not the right place to bring this up. Thank you!